static/app/views/replays/list/listContent.spec.tsx

```diff
@@ -5,7 +5,6 @@ import {render, screen, waitFor} from 'sentry-test/reactTestingLibrary';
 
 import type {Organization as TOrganization} from 'sentry/types';
 import useDeadRageSelectors from 'sentry/utils/replays/hooks/useDeadRageSelectors';
-import useReplayList from 'sentry/utils/replays/hooks/useReplayList';
```

Comment:
Is this hook still used?

Reply:
yes, for now :(

it's called by the Issue>Replay list and Txn>Replay list. They'll each need some care to rewrite things